app/services/create_proposed_organisation_edit.rb

```diff
@@ -36,6 +36,7 @@ def merge_in_non_published_fields
 
   def send_email_to_superadmin_about_org_edit
     superadmin_emails = user_klass.superadmins.pluck(:email)
```

Comment:
@tansaku wrote: The whole refactoring is actually motivated by these lines in the `create_proposed_organisation_edit_spec`:

`allow(user_klass) to receive_message_chain(:superadmins, :pluck).and_return(:admins)`
`allow(mailer_class).to receive_message_chain :edit_org_waiting_for_approval, :deliver_now`

I just wanted to know how to spot these code smells when you look at code.